docs/resources/function_cron.md

```diff
@@ -8,7 +8,7 @@ page_title: "Scaleway: scaleway_function_cron"
 Creates and manages Scaleway Function Triggers. For the moment, the feature is limited to CRON Schedule (time-based).
 
 For more information consult
-the [documentation](https://www.scaleway.com/en/docs/compute/functions/api-cli/fun-uploading-with-serverless-framework/#configuring-events)
+the [documentation](https://www.scaleway.com/en/docs/serverless/containers/api-cli/cont-uploading-with-serverless-framework#configuring-events)
```

Comment:
New link does not work for me. It send me to serverless framework repo but no on cron doc